package.json

```
@@ -26,6 +26,7 @@
     "@react-aria/focus": "^3.5.0",
     "@react-aria/interactions": "^3.7.0",
     "@react-aria/menu": "^3.3.0",
+    "@react-aria/numberfield": "~3.0.1",
```

might want to keep it with the same style as the others
```suggestion
    "@react-aria/numberfield": "^3.0.1",
```